### ISSUE TITLE
Do you have a French translation of Tanakh or at least Houmach?

### ISSUE THREAD

**Issue by benjaminb10** [Sefaria/Sefaria-Export#10]
Do you have a French translation of Tanakh or at least Houmach?

**Comment by nsantacruz** [comment]
Yes, have a full French translation of Tanakh. You can read it in the sidebar like this: https://www.sefaria.org.il/Genesis.1.1?lang=he&vside=La_bible_d%27Andr%C3%A9_Chouraqui_[fr]&with=Version%20Open&lang2=he

Or you can read it in the main panel like this:
https://www.sefaria.org.il/Genesis.1.1?ven=La_bible_d%27Andr%C3%A9_Chouraqui_[fr]&lang=bi&aliyot=0

**Comment by benjaminb10** [comment]
Thanks,

Is it accessible via the API?
Where can I find it in text in the repository?

Moreover, sorry to tell, but the translation seems not so good. Do you have the source?

Regards.
